src/features/presentation/pages/presentation/presentation.page.html

```diff
@@ -22,7 +22,7 @@ <h1 class="fw-bold mb-4">Qu'est-ce que la cartographie nationale des lieux d'inc
               <span role="img" class="ri-arrow-down-line me-2 small" aria-hidden="true"></span>
               Accéder aux webinaires
             </button>
-            <a [routerLink]="['/', 'orientation']" class="btn btn-discover rounded-0">Orienter un bénéficiaire</a>
+            <a [routerLink]="['/', 'orientation']" tabindex="0" class="btn btn-discover rounded-0">Orienter un bénéficiaire</a>
```

Comment:
```suggestion
            <a [routerLink]="['/', 'orientation']" class="btn btn-discover rounded-0">Orienter un bénéficiaire</a>
```
Je n'ai pas compris à quoi sert le `tabindex="0` ici